js/wizard/configModel.js

```diff
@@ -33,6 +33,60 @@ OCA = OCA || {};
 		 * @param {OCA.LDAP.Wizard.WizardDetectorQueue} detectorQueue
 		 */
 		init: function (detectorQueue) {
+			this.translations = {
```

Comment:
I'm not sure if this could be confused with language translations...

Reply:
renamed to `mappings`